rendercv/cli/printer.py

```diff
@@ -84,6 +84,22 @@ def start_a_step(self, step_name: str):
             f"{self.current_step_name} has started."
         )
 
+    def update_total_steps(self, total_steps: int):
```

Comment:
See https://github.com/sinaatalay/rendercv/pull/166#discussion_r1740264333. We probably don't need this.

Reply:
Deleted the function